projects/charger/src/main.c

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
/* */ style comments are not accepted.  Take a look at here: https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards